### DISCUSSION TITLE
Boot menu entries from BLS /boot/loader/entries

### DISCUSSION THREAD

**Discussion by beremour** [zbm-dev/zfsbootmenu#522] [Questions]
I've downloaded EFI image "https://get.zfsbootmenu.org/efi". It boots fine. But how can I get Boot Menu from BLS-ready configs from /boot/loader/entries/  ? Directory /boot/loader/entries/ resides on ZFS pool SYSTEM and filesystem SYSTEM/BOOT

**Comment by ahesford** [comment]
You can't. ZFSBootMenu doesn't support BLS configurations.
